README.md

```diff
@@ -59,6 +59,14 @@ https://www.npmjs.com/package/do_username
 
 `npm install do_username`
 
+### Python
+
+[README](python/README.md)
+
+https://pypi.org/project/do-username/
+
+`pip3 install do-username`
```

Comment:
Should this be `pip3 install do-username` or `pip3 install do_username`. Both work, but I guess `pip3 install do_username` would be more consistent?

Reply:
We should use `do_username` wherever possible for consistency, please :)